repose-aggregator/components/filters/add-header/src/main/resources/META-INF/schema/config/add-header.xsd

```diff
@@ -30,8 +30,8 @@
 
     <xs:complexType name="AddHeadersConfig">
         <xs:all>
-            <xs:element name="request" type="HttpMessage"/>
-            <xs:element name="response" type="HttpMessage"/>
+            <xs:element name="request" type="HttpMessage" minOccurs="0"/>
+            <xs:element name="response" type="HttpMessage" minOccurs="0"/>
```

Comment:
I totally agree.  Do you know of an easy way to do that?  I only found two unideal ways of doing it on the web.  The less insane option was to do an assertion.  I vaguely remember seeing some other filter's doc mention having such a case of not configuring anything and the filter just being a "pass through", so I figured there was precedent.  Thoughts?